src/pages/ReimbursementAccount/ReimbursementAccountPage.js

```diff
@@ -441,7 +441,7 @@ class ReimbursementAccountPage extends React.Component {
                         subtitle={policyName}
                         onBackButtonPress={() => Navigation.goBack(ROUTES.SETTINGS_WORKSPACES)}
                     />
-                    <View style={[styles.m5, styles.flex1]}>
+                    <View style={[styles.mh5, styles.mv3, styles.flex1]}>
```

Comment:
@Pujan92 Why is the margin only added to the error message? I think the whole page needs it.

Reply:
Other sections already have the required padding with the `pageWrapper`, only errorText `View` remains so applied here.

Reply:
Exactly, it looks off to me, but if you don't mind maybe you can take a screenshot and tag in @shawnborton to confirm? I think we're just trying to keep things consistent across all screens so wherever we spot it is worth fixing, I think!

Reply:
Yup, that makes sense to me!